vispy/visuals/polygon.py

```diff
@@ -82,47 +97,3 @@ def border_color(self):
     def border_color(self, border_color):
         self._border_color = Color(border_color)
         self._update()
-
-    def _update(self):
-        self.data = PolygonData(vertices=np.array(self._pos, dtype=np.float32))
-        if self._pos is None:
-            return
-        if not self._color.is_blank:
-            pts, tris = self.data.triangulate()
-            self.mesh.set_data(vertices=pts, faces=tris.astype(np.uint32),
-                               color=self._color.rgba)
-        if not self._border_color.is_blank:
-            # Close border if it is not already.
-            border_pos = self._pos
-            if np.any(border_pos[0] != border_pos[1]):
-                border_pos = np.concatenate([border_pos, border_pos[:1]], 
-                                            axis=0)
-            self.border.set_data(pos=border_pos,
-                                 color=self._border_color.rgba, 
-                                 width=self._border_width,
-                                 connect='strip')
-        self.update()
-
-    def set_gl_options(self, *args, **kwargs):
-        self.mesh.set_gl_options(*args, **kwargs)
-
-    def update_gl_options(self, *args, **kwargs):
-        self.mesh.update_gl_options(*args, **kwargs)
-
-    def draw(self, transforms):
-        """Draw the visual
-
-        Parameters
-        ----------
-        transforms : instance of TransformSystem
-            The transforms to use.
-        """
-        if self._pos is None:
-            return
-        if not self._color.is_blank:
-            gloo.set_state(polygon_offset_fill=True, 
```

Comment:
`polygon_offset_fill=True` needs to be applied to the mesh at some point (probably either init or _prepare_draw)